utils/node-fuzzer/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "node-fuzzer"
```

Comment:
In fact it's more likely runtime-fuzzer, isn't it?

Reply:
done, renamed everywhere